app/forms/support_interface/application_forms/change_course_choice_form.rb

```diff
@@ -0,0 +1,27 @@
+module SupportInterface
+  module ApplicationForms
+    class ChangeCourseChoiceForm
+      include ActiveModel::Model
+
+      attr_accessor :application_choice_id, :provider_code, :course_code, :study_mode, :site_code, :accept_guidance, :audit_comment_ticket
+
+      validates :provider_code, :course_code, :study_mode, :site_code, :accept_guidance, :audit_comment_ticket, presence: true
+      validates :audit_comment_ticket, format: { with: /\A((http|https):\/\/)?(www.)?becomingateacher.zendesk.com\/agent\/tickets\// }
```

Comment:
Consider making the regex a constant, easier to amend, test and slightly more efficient.

Reply:
We do seem to be using this validation in a number of places, so consider adding a custom reusable validator.